### PR TITLE
More timer debugging for the debug gods.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
Records what time a timer is spent for debug outputting

A runtime was made to use `get_timer_debug_string`.

`get_timer_debug_string` reports rather or not something has a callback at that moment.

And I made an undocumented change to timer's New

### CODE REVIEW

**Comment by RemieRichards** [comment]
@MrStonedOne do you mean being able to add it to the last commit, or add more commits? (re: last commit message)

**Comment by MoreRobustThanYou** [comment]
> patch-420

insert shitty 420 meme here

File: code/controllers/subsystem/timer.dm [hunk 1]
```diff
@@ -135,9 +135,11 @@ SUBSYSTEM_DEF(timer)
 	. = "Timer: [TE]"
 	. += "Prev: [TE.prev ? TE.prev : "NULL"], Next: [TE.next ? TE.next : "NULL"]"
 	if(TE.spent)
-		. += ", SPENT"
+		. += ", SPENT([TE.spent])"
 	if(QDELETED(TE))
 		. += ", QDELETED"
+	if(!timer.callBack)
```
Comment (Cyberboss) [inline code/controllers/subsystem/timer.dm]: TE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by MrStonedOne** [comment]
@RemieRichards add it to the last commit

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**